utils/text_formatting.jsx

```diff
@@ -49,13 +49,15 @@ export function formatText(text, inputOptions) {
     }
 
     let output = text;
-
     const options = Object.assign({}, inputOptions);
+    const hasPhrases = (/"([^"]*)"/).test(options.searchTerm);
```

Comment:
Should we be using the xregexp like the others here?

Reply:
That's a good question. I am not familiar with XRegExp. That expression is practically the same that appears in `parseSearchTerms` which I re-used.

https://github.com/mattermost/mattermost-webapp/blob/cbf67e07f5390540d0cb74b51a7ce53cd61ccd14/utils/text_formatting.jsx#L410

Reply:
XRegExp is only needed if you're using regex features that aren't supported natively in JS. This is fine using a regular regular expression